demos/universal-sentence-encoder/README.md

```diff
@@ -0,0 +1,127 @@
+# Using inputs data in string format with universal-sentence-encoder model
+
+
+## Download the model
+
+In this experiment we are going to use a TensorFlow model from [tfhub.dev ](https://tfhub.dev/google/universal-sentence-encoder-multilingual/3).
+
+```bash
+curl --create-dir  https://storage.googleapis.com/tfhub-modules/google/universal-sentence-encoder-multilingual/3.tar.gz -o universal-sentence-encoder-multilingual/1/3.tar.gz
+tar -xzf universal-sentence-encoder-multilingual/1/3.tar.gz -C universal-sentence-encoder-multilingual/1/
+rm universal-sentence-encoder-multilingual/1/3.tar.gz
+tree universal-sentence-encoder-multilingual/
+
+universal-sentence-encoder-multilingual/
+└── 1
+    ├── assets
+    ├── saved_model.pb
+    └── variables
+        ├── variables.data-00000-of-00001
+        └── variables.index
+
+```
+Temporarily OVMS requires the model in a frozen graph format so a similar structure is to be created.
+The graph format represents the input and output as a tensorname so we will apply a `mapping_tensor.json` to adjust the model server names.
+```json
+{
+       "inputs":{ 
+          "Func/StatefulPartitionedCall/input/_0": "inputs"
+       },
+       "outputs":{
+          "Func/StatefulPartitionedCall/output/_500":"ouputs"
+       }
+}
+``` 
+
+```
+tree universal-sentence-encoder-multilingual-frozen
+universal-sentence-encoder-multilingual-frozen
+└── 1
+    ├── mapping_config.json
+    └── muse.pb
+```
+Note: frozen graph is created using [freeze_graph.py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/tools/freeze_graph.py) script.
+
+
+## Build OVMS with CPU extension library for sentencepiece_tokenizer layer
+
+Model universal-sentence-encoder-multilingual includes a layer SentencepieceTokenizer which is not supported by OpenVINO at the moment. It can be however implemented using a [cpu extention](https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/custom_operations/user_ie_extensions/sentence_piece), which is a dynamic library performing the execution of the model layer.
```

Comment:
```suggestion
Model universal-sentence-encoder-multilingual includes a layer SentencepieceTokenizer which is not supported by OpenVINO at the moment. It can be however implemented using a [CPU extension](https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/custom_operations/user_ie_extensions/sentence_piece), which is a dynamic library performing the execution of the model layer.
```